bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/parallacticangle/PositionAnglePanel.scala

```diff
@@ -221,15 +217,26 @@ class PositionAnglePanel[I <: SPInstObsComp with PosAngleConstraintAware,
   }
 
   /**
+    * The actual copying of a given pos angle to the data object.
+    * This is done explicitly to avoid overwriting based on minor differences in double precision and to avoid
+    * adding 180 to the position angle, which causes issues.
+    **/
+  private def setInstPosAngle(newAngleDegrees: Double): Unit = editor.foreach { e =>
+    val Precision = 0.005
+    val oldAngleDegrees = e.getDataObject.getPosAngle
+    if (Math.abs(oldAngleDegrees - newAngleDegrees) >= Precision && Math.abs(Math.abs(oldAngleDegrees - newAngleDegrees) - 180) >= Precision) {
+      e.getDataObject.setPosAngle(newAngleDegrees)
```

Comment:
Could you explain a bit more here?  From the point of view of the guide probe range, often there is a big difference when adding 180 degrees.  For example with GMOS-OI, the guide stars reachable at 0 degrees are not the same as those at 180.  But perhaps this setter is only called by the parallactic angle calculation?